packages/framework-provider-aws-infrastructure/test/infrastructure/stacks/application-stack.test.ts

```diff
@@ -116,6 +110,7 @@ describe('the application stack builder', () => {
     // Lambdas
     expect(numberOfLambdas).to.equal(4)
     expect(appStack.tryFindChild(preSignUpValidator)).not.to.be.undefined
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
At this point, I'd remove the `any` and would type `lambda` as `lambda: { environment: Record<string, string> }`

Reply:
I tried, but it's not that easy in practice 😬